src/styles/toolbar.styl

```diff
@@ -1,6 +1,22 @@
 @require 'cozy-ui'
 
-.pho-content-toolbar
-  margin-left auto
-  .coz-btn:last-child
-    margin-right 0
+.pho-toolbar
+    margin-left auto
+
+    .coz-btn:last-child
+        margin-right 0
+
+    a
+        &.pho-action-select
```

Comment:
I don't get what you're trying to do here and the next selector below, it seems like the same target but with 2 different ways of selection. 😕